tests/shared/process-event.ts

```diff
@@ -235,9 +235,9 @@ export const createProcessEventTests = (
         )
 
         if (database === 'clickhouse') {
-            expect(queryCounter).toBe(11 + 14 /* event & prop definitions */)
+            expect(queryCounter).toBe(10 + 14 /* event & prop definitions */)
```

Comment:
👀 

which query doesn't happen anymore?

Reply:
We don't do `shouldSendWebhooks` anymore 🎉